README.md

```diff
@@ -87,12 +88,13 @@ A curated list of awesome open source workflow engines
 * [Camunda](https://camunda.com) [![Stars](https://img.shields.io/github/stars/camunda/camunda-bpm-platform.svg)](https://github.com/camunda/camunda-bpm-platform) - BPMN-based workflow engine that can be embedded as java library (e.g. Spring Boot) or used standalone, including a graphical modeler and operations tooling.
 * [Captain](https://github.com/LiveRamp/captain) [![Stars](https://img.shields.io/github/stars/LiveRamp/captain.svg)](https://github.com/LiveRamp/captain) - Distributed, light-weight java workflow engine for a microservice architecture.
 * [CoreWF](https://github.com/UiPath/corewf) [![Stars](https://img.shields.io/github/stars/UiPath/corewf.svg)](https://github.com/UiPath/corewf) - WF runtime ported to work on .NET Core
+* [ctrl+flow](https://github.com/ctrlflowjs/ctrlflow) [![Stars](https://img.shields.io/github/stars/ctrlflowjs/ctrlflow.svg)](https://github.com/ctrlflowjs/ctrlflow) - Fullstack JS framework to embed visual programming interfaces in web apps, with durable background processing. Delivers an intuitive UX that turns non-technical users into collaborators and reduces load on dev teams.
```

Comment:
no point in adding it twice

Reply:
gotcha, wasn't sure, it fit both categories and saw other options listed in more than one place.